sway-core/Cargo.toml

```diff
@@ -14,6 +14,7 @@ selector-debug = ["structopt", "hex"]
 [dependencies]
 derivative = "2.2.0"
 either = "1.6"
+dirs = "3.0.2"
```

Comment:
Hmm. Could you sort alphabetically and use `"3.0"` instead (unless there's a specific reason for `"3.0.2"`?

Reply:
Sure. `".3.0.2"` is specified by `forc`, so I assumed that requiring the exact same version would be a good idea.

Maybe we should add a CI check that ensures alphabetic sorting so humans don't need to check manually?

Reply:
> Maybe we should add a CI check that ensures alphabetic sorting so humans don't need to check manually?

The issue is that there's no simple canonical formatter to alphabetize/check dependencies. https://github.com/rust-lang/rustfmt/issues/4091

If you want to roll up your sleeves and figure something out, it would be amazing! It should absolutely be part of our CI pipeline.